.gitignore

```diff
@@ -1,3 +1,7 @@
 build/
-*.user
+*.user*
```

Comment:
@AGritsevich yes, to see clear git. As you know moust of the developers use qtcreator